### ISSUE TITLE
Post Excerpt Block: Allow inline html tags

### ISSUE THREAD

**Issue by supernovia** [Automattic/wp-calypso#74675]
### What

The Post Excerpt has traditionally allowed limited HTML formatting (strong, em, etc), but it's being stripped by the Post Excerpt Block. Could we have an option to keep formatting?

### Why

People like being able to format their excerpts by hand for many reasons. [Here's one case](https://wordpress.com/forums/topic/html-tags-suggested-in-the-manual-excerpt-documentation-are-not-visibly-applied/); our user wants to add a little extra meta to the excerpt, and it works fine unless they use the Post Excerpt Block for designing their site.  I'm aware of people hand-formatting their exceprts for a variety of other reasons, too: haiku, adding links, sectioning off a part for emphasis, etc. And that all gets lost if we use the Post Excerpt Block.

The workaround for now is to use some other posts-related block, but they don't offer as much control as the query loop.

### How

The Post Exceprt Block has [its own text-formatting features](https://wordpress.org/documentation/article/post-excerpt-block/), and I wonder if that's why they're being stripped. Maybe we could only strip tags if one of those features is utilized, and otherwise display the inline HTML as entered?

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept
- Shared Feature request in Core: https://github.com/WordPress/gutenberg/issues/49449

📌 Message to Author
- @supernovia not sure what your handle is for that repo, so I couldn't tag you, but please add a comment to take ownership of that, I think your suggestions are great, and you may get follow up questions!

**Comment by cuemarie** [comment]
From the Core report:

> I think the problem is a bug. The post excerpt should not remove HTML tags by default, but this block removes HTML tags on the front end. Below are the results of the test on Twenty Twenty One.

--> https://github.com/WordPress/gutenberg/issues/49449#issuecomment-1489587593

📌 ACTIONS
- Relabeled as a bug, and moved to Non-Urgent

📌 Message to HEs
- Please track any interactions about this issue here on this report, not the Gutenberg issue. Thanks!

**Comment by lizkarkoski** [comment]
7123938-zen 

Behavior - the user is unable to use html to create line breaks in the post excerpts. 
<img width="277" alt="Screenshot 2023-11-06 at 5 20 30 AM" src="https://github.com/Automattic/wp-calypso/assets/16867830/e6de5e94-0f74-49d2-bb96-6ab77850ab47">
<img width="341" alt="Screenshot 2023-11-06 at 5 20 52 AM" src="https://github.com/Automattic/wp-calypso/assets/16867830/acd37c0a-8e70-4dd8-b982-02e8a0f57773">

**Comment by Daife222** [comment]
Html in excerpt is/was for me very useful. 
Please upgrade the bug resolution urgency from "non-urgent".
Wordpress 6.4.2.
Thanks

**Comment by Peter-HVD** [comment]
I agree with @Daife222 - this should be marked as an urgent bug because it is long-established WordPress functionality that HTML tags are not stripped from custom excerpts. I have just built a site with a block theme for the first time, and my client is confused and complains that it is not working as their previous theme did. It's not acceptable to have to say "the WordPress people have taken an arbitrary decision to change the way it works and there's nothing I can do about it".

**Comment by Mfarroq1** [comment]
7868097-zen
